faust/tables/objects.py

```diff
@@ -65,20 +65,18 @@ class ChangeloggedObjectManager(Store):
     data: MutableMapping
 
     _storage: Optional[StoreT] = None
-    _dirty: Set
```

Comment:
Wow, this `_dirty` attr existed since the very first commit of this file. Looks like it never got replaced by anything better. I'm looking forward to this PR changing that.